test/BlobStorageClient.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const request = require('request');
+const sinon = require('sinon');
+const chai = require('chai');
+const blobStorageClient = require('../src/clients/storage/BlobStorageClient');
+chai.should();
+const chaiAsPromised = require('chai-as-promised');
+chai.use(chaiAsPromised);
+
+describe('Tests for BlobStorageClient.js', function() {
```

Comment:
The reason why I used function here was because of http://mochajs.org/#arrow-functions
which stated: `Passing arrow functions (“lambdas”) to Mocha is discouraged. Due to the lexical binding of this, such functions are unable to access the Mocha context.`